setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-docker build -t rgardler/acs-logging-test-base .
+docker build -t acs-logging-test-base .
```

Comment:
Why change the publisher? It makes some sense since this is open source code, but only changing for one of the images is likely to confuse.